src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/cachemanager/PrimitiveCacheNode.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.tools.coveragemodel.cachemanager;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class represents a primitive cache node (just stores a value and does not perform any computation)
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class PrimitiveCacheNode extends CacheNode {
+
+    private static List<String> EMPTY_LIST = new ArrayList<>();
```

Comment:
This probably doesn't need to be extracted (since it is used only in the constructor).  Can be final if you want to keep it.  Also, should this be `Collections.emptyList()` instead of `new ArrayList<>()`?